src/main.rs

```diff
@@ -216,6 +216,21 @@ fn check_for_existing_files(replacements: &Vec<Rename>) -> anyhow::Result<()> {
     Ok(())
 }
 
+fn check_input_files(input_files: &Vec<String>) -> anyhow::Result<()> {
+    let nonexisting_files : Vec<_> = input_files.iter().filter(|input_file| !Path::new(input_file).exists()).collect();
+
+    if !nonexisting_files.is_empty() {
+        println!("The following input files do not exist:");
+        for file in nonexisting_files {
+            println!("{}", Colour::Green.paint(file));
```

Comment:
What about red or yellow to better show there is a problem?

Reply:
Yes, you are right. I used the same color for the files printed when checking for existing files. Maybe these should also be in red/yellow?